src/PKSim.Infrastructure/ORM/Repositories/OntogenyRepository.cs

```diff
@@ -99,6 +118,56 @@ public double OntogenyFactorFor(Ontogeny ontogeny, string containerName, double?
          return _interpolation.Interpolate(allOntogenies.Select(x => new Sample(x.PostmenstrualAge, factorRetriever(x))), postmenstrualAgeInYearsFor(age, gestationalAge));
       }
 
+      public DistributedTableFormula DataRepositoryToDistributedTableFormula(DataRepository ontogenyDataRepository)
```

Comment:
This is existing code that I moved from presenter here so that I could reuse. In short, it helps creating a table from an ontogeny selection